### DISCUSSION TITLE
GitHub Careers

### DISCUSSION THREAD

**Discussion by TheHellTower** [community/community#45677] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I'm not sure it's the right place to ask so if it's not please let me know as soon as you see it, thanks.
So today I seen GitHub have a [Careers](https://github.com/about/careers) page but "no position are open", is there any way to know more about the potential future open positions ? I might be interested,

Regards.

**Comment by jge162** [comment, marked as answer]
@TheHellTower Microsoft just laid off 10k people probably will be awhile. [News Link here](https://www.npr.org/2023/01/18/1149739148/microsoft-slashes-10-000-jobs-the-latest-in-a-wave-of-layoffs)

**Reply by TheHellTower** [reply]
> Microsoft just laid off 10k people probably will be awhile
Oh perfect I will be patient then 😂

**Comment by Negi97Mohit** [comment]
I got an email for TA, I gave the TA and its been more than two weeks since I last heard from them.
I passed the TA, but still no response, joining github would be like a dream come true tbh,
plus its my BDay on 22 Dec so this thing would just may aswell be my gift.

How do i connect with github?? and explain em my dillema...

**Reply by lastforkbender** [reply]
Not sure if you are using a translator or a bot...
